### DISCUSSION TITLE
Holodeck instructions in the read me

### DISCUSSION THREAD

**Discussion by cankisagun** [enigmampc/secret-contracts-guide#14] [General]
@lauraweindorf cc @levackt does it make more sense to add holodeck information in the setup process or after Alice has gone through the full tutorial and wrote some contracts first?
cc @toml01

**Comment by levackt** [comment]
Oh, I think after, start local.

**Reply by cankisagun** [reply]
After start local or at the very end?

**Comment by levackt** [comment]
At the very end, when you're ready to progress to a testnet.

And link to the section at the start along with other sections, so users
not interested or able to run locally and setup the connection to testnet
instead.
